src/summary/composer.rb

```diff
@@ -115,6 +119,21 @@ def get_resource_services(installed_services, resource_id)
       return collection
     end
 
+    def get_resource_deployment_name(provisioned_resource)
+      output = ""
+      if provisioned_resource.get_resource().respond_to?(:get_tags)
+        deployment_name = provisioned_resource.get_resource().get_tags()["dxDeploymentName"].to_s
```

Comment:
@noahmmcgivern the deployment name is accessible through the commandline provider, we shouldn't go get it from a resource tag.
https://github.com/newrelic-experimental/deployer/blob/main/src/command_line/provider.rb#L59